Src/AutoFixture.xUnit.net2.UnitTest/Scenario.cs

```diff
@@ -72,6 +72,42 @@ public void InlineAutoDataSuppliesDataSpecimensOnlyForNonProvidedValues(string s
             Assert.NotEqual("bar", s3);
         }
 
+        // This test and its associated types is used to document one of the
+        // InlineAutoDataAttribute constructor overloads.
+        [Theory]
+        [MyCustomInlineAutoData(1337)]
+        [MyCustomInlineAutoData(1337, 7)]
```

Comment:
How about removing the second parameter altogether? I would probably also rename the custom auto data attribute to be more specific:

```csharp
[Theory]
[AllNumbersAre42InlineAutoData(1337)]
[AllNumbersAre42InlineAutoData(1337, 42)]
public void CustomInlineDataSuppliesExtraValues(int x, int y)
{
    Assert.Equal(1337, x);
    Assert.Equal(42, y);
}
```

I think this makes the intent of replacing customized types with specific values a little clearer.